tests/server/models/review-test.js

```diff
@@ -0,0 +1,56 @@
+var sinon = require('sinon');
+var expect = require('chai').expect;
+
+var Sequelize = require('sequelize');
+
+var db = require('../../../server/db');
+
+var Review = db.model('review');
+
+describe('Review model', function () {
+
+    beforeEach('Sync DB', function () {
+        return db.sync({ force: true });
+    });
+
+    it('creates a review',
+        function (){
+            return Review.create({stars: 3, content: 'This is the best. 1234567890.'})
+            .then(function(response){
+                expect(response.stars).to.exist;
+            })
+        }
+    );
+
+    it('Stars # must be less than 6', function () {
+        return Review.create({stars: 7, content: 'This is terrible. 1234567890.'})
```

Comment:
Given that you're just checking validation, I would look into the [`Instance.validate()`](http://docs.sequelizejs.com/en/latest/api/instance/#validateoptions-promiseerrorsvalidationerrorundefined) method in the Sequelize docs which will be cleaner to test while accomplishing the same goal.